### PR TITLE
Navbar inital style added

### PR DESCRIPTION
**Pull request by pablovillasana**

### CODE REVIEW

**Comment by gonzalo-plaza** [comment]
@PVillasana The translation button is no vertical centered with her box.
translation-button1

I think is better if the translation button is vertical centered with her box.
translation-button2

**Comment by pablovillasana** [comment]
> @PVillasana The translation button is no vertical centered with her box.
> translation-button1
> 
> I think is better if the translation button is vertical centered with her box.
> translation-button2

@gonzalo-plaza Dont worry, is already changed and it will be upload on the future

File: src/components/common/LastReviewed.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<template>
+    <vs-tooltip bottom>
+        {{ lastVersion }}
+        <template #tooltip
```
Comment (Andresws12) [inline src/components/common/LastReviewed.vue:4]: @PVillasana Fix format manually please

File: src/components/common/TranslationButton.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<template>
+    <vs-button active class="translation-button" @click="toggleLanguage">{{
```
Comment (Andresws12) [inline src/components/common/TranslationButton.vue:2]: @PVillasana Fix format manually please

File: src/styles/vuesax-overrides.scss [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+    --vs-primary: 0, 0, 0;
+    --vs-success: 23, 201, 100;
+    --vs-danger: 242, 19, 93;
+    --vs-warn: 254, 130, 0;
+    --vs-dark: 36, 33, 69;
+}
+
+// Common
 body {
     font-family: 'Poppins-Regular';
```
Comment (Andresws12) [inline src/styles/vuesax-overrides.scss]: @PVillasana Why, are not you using the font variable?

Comment (pablovillasana) [reply src/styles/vuesax-overrides.scss]: @Andresws12 You are telling me to use the library variable to override the default font?

Comment (Andresws12) [reply src/styles/vuesax-overrides.scss]: @PVillasana No, You declared a new variable and you didn't use this

Comment (Andresws12) [reply src/styles/vuesax-overrides.scss]: image

Comment (pablovillasana) [reply src/styles/vuesax-overrides.scss]: @Andresws12 Ok, you refereed to the fallback fonts variables

**Pull request merged**